### ISSUE TITLE
Feature request: allow selectors to select elements in html at `feed.config.link`

### ISSUE THREAD

**Issue by DCsunset** [wezm/rsspls#56]
Hello,

Thanks a lot for this great project!

When I'm trying to use this tool, I found that some websites only provide a list of titles and links in the overview page, which makes it impossible to generate items like summaries, enclosures, and etc.

It would be great if rsspls could support fetching the entry's own web page (defined in `feed.config.link`) and selecting elements in it.

**Comment by wezm** [comment]
Just to confirm, for a page of links like: https://lobste.rs/ and a `feed.config.link` selector like `.story .u-url` it would fetch each of those URLs in order to match `feed.config.summary` etc?

**Comment by DCsunset** [comment]
Yes that's what I'm thinking of. The selectors can work the same way but on a different URL for each entry.
